README.md

```diff
@@ -29,20 +29,20 @@ For setting ScalarDB properties, see also the related documents [here](https://g
 
 
 ```console
-java -jar scalardb-schema-loader-<version>.jar --config /path/to/scalardb.properties -f tpcc-schema.json
+java -jar scalardb-schema-loader-<version>.jar --config /path/to/scalardb.properties -f tpcc-schema.json --coordinator
```

Comment:
In the data loading step after schema creation, I faced the following error.
```java
com.scalar.db.exception.transaction.CommitException: The specified table is not found: coordinator.state
        at com.scalar.db.transaction.rpc.GrpcTransactionOnBidirectionalStream.throwIfErrorForCommit(GrpcTransactionOnBidirectionalStream.java:279)
        at com.scalar.db.transaction.rpc.GrpcTransactionOnBidirectionalStream.commit(GrpcTransactionOnBidirectionalStream.java:257)
        at com.scalar.db.transaction.rpc.GrpcTransaction.commit(GrpcTransaction.java:77)
        at com.scalar.db.benchmarks.tpcc.TpccLoader.insert(TpccLoader.java:287)
        at com.scalar.db.benchmarks.tpcc.TpccLoader.lambda$load$0(TpccLoader.java:315)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:833)
```

So, I think we need to create `coordinator.state` table when we create schemas for TPC-C.